tests/multi_bluetooth/ble_gap_advertise.py

```diff
@@ -37,18 +37,21 @@ def irq(ev, data):
         nonlocal finished, adv_types, adv_data
         if ev == _IRQ_SCAN_RESULT:
             if data[1] == BDADDR:
-                adv_types.add(data[2])
-                if adv_data is None:
-                    adv_data = bytes(data[4])
-                else:
-                    if adv_data != data[4]:
-                        adv_data = "MISMATCH"
+                if data[2] in (0, 2):
```

Comment:
This (getting scan responses) was exactly the issue I originally had with BTstack on PYBD, which is now fixed by the latest BTstack update.  Did you use the latest master?  Or was there some other reason scan responses are passed through to here?

Reply:
Yes, latest master, but I may have forgotten to update the submodule. I'll check tomorrow and if that's not it I'll find out for sure.